package.json

```diff
@@ -340,7 +340,7 @@
     "style-loader": "0.23.1",
     "styled-components": "5.3.9",
     "terser": "5.16.2",
-    "trezor-connect": "8.2.6-extended",
+    "trezor-connect": "8.2.12",
```

Comment:
I think we need the `8.2.12-extended` version here which includes dependencies necessary for USB transport.

Reply:
👍🏻 makes sense - just updated it to `8.2.12-extended`